src/components/calcite-alert/calcite-alert.tsx

```diff
@@ -92,6 +92,17 @@ export class CalciteAlert {
     this.requestedIcon = setRequestedIcon(StatusIcons, this.icon, this.color);
   }
 
+  @Watch("autoDismissDuration")
+  updateDuration(newDuration: string, prevDuration: string): void {
+    if (this.autoDismiss && prevDuration !== newDuration && this.autoDismissTimeout) {
```

Comment:
You can drop `prevDuration !== newDuration` since it will always be true. Watchers only get invoked when the new and old values are different.

Reply:
nice catch 🏈